animata/card/feature-card.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { Check } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+
+interface FeatureCardProps {
+  title?: string;
+  placeholder?: string;
+  buttonText?: string;
+}
+
+export default function FeatureCard({
+  title = "Want to read the rest?",
+  placeholder = "justin@buttondown.email",
+  buttonText = "Subscribe for $5/mo",
+}: FeatureCardProps) {
+  const [isButtonHovered, setIsButtonHovered] = useState(false);
```

Comment:
You don't need to add the state for `hover`. Simply use `hover:skew-x-0 hover:skew-y-0`

Reply:
Just to clarify, the skew effect is triggered on hovering the card not just the button.

Reply:
> Just to clarify, the skew effect is triggered on hovering the card not just the button.

okay I saw video, there skewing card  when hovering btn, so I done I like sir

Reply:
@hari should i change now or okay, skew resetting after button hover looks more good i feel...
If not required i will change sir ... please let me know

Reply:
@Prithwi32 you can change it.

Reply:
Okay

Reply:
Done sir @hari 
check this: 

https://github.com/user-attachments/assets/c60c32f8-be64-4ab8-bcd7-eca2c3b21695